nft_ingester/Cargo.toml

```diff
@@ -28,7 +28,10 @@ digital_asset_types = { path = "../digital_asset_types", features = ["json_types
 uuid = "1.0.0"
 async-trait = "0.1.53"
 num-traits = "0.2.15"
```

Comment:
Can probably add mpl-candy-machine now that Austin updated the Anchor version in MPL for candy machine